drivers/regulator/regulator_fixed.c

```diff
@@ -59,9 +59,33 @@ static int regulator_fixed_disable(const struct device *dev)
 	return gpio_pin_set_dt(&cfg->enable, 0);
 }
 
+static unsigned int regulator_fixed_count_voltages(const struct device *dev)
+{
+	int32_t min_uv;
+
+	return !regulator_common_get_min_voltage(dev, &min_uv);
+}
+
+static int regulator_fixed_list_voltage(const struct device *dev,
+					unsigned int idx,
+					int32_t *volt_uv)
+{
+	if (idx != 0) {
+		return -EINVAL;
+	}
+
+	if (regulator_common_get_min_voltage(dev, volt_uv)) {
```

Comment:
`if (regulator_common_get_min_voltage(dev, volt_uv) < 0)`

Reply:
I see no reason to do it, only `0` ret value means successful execution of function according description

Reply:
ditto